label_sync/labels.yaml

```diff
@@ -41,22 +41,22 @@ labels:
     addedBy: googlebot
   - color: d455d0
     description: "TODO, see [kubernetes/test-infra#7251](https://github.com/kubernetes/test-infra/issues/7251)"
-    name: close/duplicate
+    name: triage/duplicate
```

Comment:
We ought to use the deprecated field for the old name, that way we'll rename existing labels rather than create new ones.

Reply:
Thanks @fejta Did you mean to add this by use deprecated field? Thanks!
```previously:
      - name: close/duplicate
      - name: ‘close/duplicate (deprecated - do not use)' ```

Reply:
nope, it's the "previously" field he is talking about. Line 123 ("help wanted") has an example.